libcloud/dns/types.py

```diff
@@ -69,11 +69,9 @@ class RecordType(object):
     TXT = 'TXT'
     PTR = 'PTR'
     SOA = 'SOA'
-    HINFO = 'HINFO'
     SPF = 'SPF'
     SRV = 'SRV'
     SSHFP = 'SSHFP'
-    PTR = 'PTR'
```

Comment:
Why was this removed?  I'm pretty sure the Rackspace RDNS stuff I added recently references it.